docs/UPGRADING.md

```diff
@@ -2,6 +2,19 @@
 
 Here you can find a list of migration guides to handle breaking changes between releases of the CLI.
 
+## Unreleased
+
+### `ParseReferences()` function change in`arduino-cli` package
```

Comment:
```suggestion
### `github.com/arduino/arduino-cli/cli/arguments.ParseReferences` function change
```

This convention is in use previously in the document for referring to packages:

https://arduino.github.io/arduino-cli/dev/UPGRADING/#githubcomarduinoarduino-clii18n-package

My proposal extends that approach to provide fully qualified symbol references, as is use with [`go doc`](https://pkg.go.dev/cmd/doc)